apps/2FH/instagram/frontend/src/components/providers/assets/settingsSVG.tsx

```
@@ -0,0 +1,7 @@
+export const SettingsSvg=()=>{
```

ESLint

[error] Filename is not in pascal case. Rename it to `SettingsSvg.tsx`.

(unicorn/filename-case)